master/propidx.html

```diff
@@ -498,6 +322,7 @@ <h1>Property Index</h1>
           <td>N/A</td>
           <td><a href="https://www.w3.org/TR/2008/REC-CSS2-20080411/media.html#visual-media-group">visual</a></td>
           <td>yes</td>
+          <td></td>
```

Comment:
In a future PR, we should specify the computed value for stop-opacity (by copying off stroke-opacity).